docs/build-on-polkadot/xcm/hrmp-channels/overview.md

```diff
@@ -0,0 +1,211 @@
+
+#### Step 2 - Create Channel Opening Extrinsic
+
+1. In Polkadot.js Apps, connect to the relay chain, navigate to the **Developer** dropdown and select the **Extrinsics** option
+
+    ![](/polkadot-ecosystem-docs-draft/images/build-on-polkadot/hrmp-channels/hrmp-channels-4.webp)
+
+2. Construct an `hrmpInitOpenChannel` extrinsic call
+
+    1. Select the **hrmp** pallet
+    2. Choose the **hrmpInitOpenChannel** extrinsic
+    3. Fill in the parameters
+        - **recipient** - parachain ID of the target chain (in this case, 2600)
+        - **proposedMaxCapacity** - max number of messages that can be pending in the channel at once
+        - **proposedMaxMessageSize** - max message size that could be put into the channel
+    4. Copy the encoded call data
+    ![](/polkadot-ecosystem-docs-draft/images/build-on-polkadot/hrmp-channels/hrmp-channels-5.webp)
+    The encoded call data for opening a channel with parachain 2600 is `0x3c00280a00000800000000001000`.
+
+#### Step 3 - Crafting and Submitting the XCM Message {: #init-crafting-and-submitting-the-xcm-message }
+
+To initiate the HRMP channel opening process, you need to create an XCM message that includes the encoded `hrmpInitOpenChannel` call data from the previous step. This message will be sent from your parachain to the relay chain.
+
+This example uses the `sudo` pallet to dispatch the extrinsic. Verify the XCM configuration of the parachain you're working with and ensure you're using an origin with the necessary privileges to execute the `polkadotXcm.send` extrinsic.
+
+The XCM message should contain the following instructions:
+
+- **WithdrawAsset** - withdraws assets from the origin's ownership and places them in the Holding Register
+- **BuyExecution** - pays for the execution of the current message using the assets in the Holding Register
+- **Transact** - execute the encoded transaction call
+- **RefundSurplus** - increases the Refunded Weight Register to the value of the Surplus Weight Register, attempting to reclaim any excess fees paid via BuyExecution
+- **DepositAsset** - subtracts assets from the Holding Register and deposits equivalent on-chain assets under the specified beneficiary's ownership
+
+In essence, this process withdraws funds from the parachain's sovereign account to the XCVM Holding Register, then uses these funds to purchase execution time for the XCM Transact instruction, executes Transact, refunds any unused execution time and deposits any remaining funds into a specified account.
+
+To submit the XCM message:
+
+1. Connect to parachain 2500
+   
+2. Construct the XCM message by filling in all required parameters. Make sure to replace the **call** field with your encoded `hrmpInitOpenChannel` call data from the previous step
+   
+3. Click the **Submit Transaction** button to dispatch the XCM message to the relay chain.
+
+![](/polkadot-ecosystem-docs-draft/images/build-on-polkadot/hrmp-channels/hrmp-channels-6.webp)
```

Comment:
It's weird that this image has direct steps 2 and 3. I understand that it's mapping the sentences above, but maybe to make it even more precise, you can do the following:

```
To submit the XCM message, ensure you are connected to parachain 2500. Then: 

1. Construct the XCM message by filling in all required parameters. Make sure to replace the **call** field with your encoded `hrmpInitOpenChannel` call data from the previous step

2. Click the **Submit Transaction** button to dispatch the XCM message to the relay chain.
```